.gitignore

```diff
@@ -4,6 +4,7 @@ __pycache__/
 *$py.class
 .vscode/
 .DS_Store
+/.DS_Store
```

Comment:
Why do we have 2 .DS_Store files?

Reply:
The first .DS_Store file ignores .DS_Store in the root folder while the second one ignores .DS_Store files inside subfolders.

Reply:
I think the second one wasn't neccessary, I have eliminated it.